MobileCenterCrashes/MobileCenterCrashes/MSCrashes.mm

```diff
@@ -614,7 +614,7 @@ - (void)startCrashProcessing {
 }
 
 - (void)processCrashReports {
-  NSError *error = NULL;
+  NSError *error = nil;
```

Comment:
this doesn't make a difference, but I'm ok with it, it's probably more intuitive.